java/src/server/games/ServerManager.java

```diff
@@ -19,29 +19,31 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
+
+import static server.plugin.PluginConfig.PluginType.PERSISTENCE;
 
 public class ServerManager implements IServerManager {
     private Map<Integer, IServer> runningServers = new HashMap<>();
     private IServerCommunicator communicator;
     private ServerModel model;
     private IPluginLoader pluginLoader;
-    private List<IPlugin> plugins = new ArrayList<>();
+    private List<AbstractPlugin> plugins = new ArrayList<>();
     private IPersistenceProvider persistenceProvider;
 
     public ServerManager() throws IOException {
         communicator = new ServerCommunicator(this);
         pluginLoader = new PluginLoader();
 
         //TODO:: fix args for real
-        // pluginCmds String must be in the format of 'name:option, name2:option' Example - mongo:100
-        // pluginDir is the directory that we want to put plugins in
-        String pluginCmds = null;
-        File pluginDir = null;
+        String pluginConfigFile = "java/" + "plugins/config.yaml";
+        String persitence = "mongo";
```

Comment:
Typo - `persistence`. So does this mean that the default persistence provider is MongoDB?